frontend/src/libs/components/protected-route/protected-route.tsx

```diff
@@ -0,0 +1,22 @@
+import { Navigate } from 'react-router-dom';
+
+import { AppRoute } from '~/libs/enums/enums.js';
+
+import { RouterOutlet, Spinner } from '../components.js';
+
+const ProtectedRoute = (): React.ReactElement | null => {
+  const isLoggedIn = Math.random();
```

Comment:
I think you need to make changes in store, and retrieve user from the store. I know that now there's  no log in functionality, but I don't like the idea to merge this approach with Math.random into development 🤔

Reply:
I did this to test how it works and created a pull request so you can see the rest of the code. It was not supposed to be added in this form to development.

If this component is needed now, then I will do as you say and update it.

Reply:
I think you need to add changes in this or, @PavloSukhinin @Druxight what are u thinking about this?

Reply:
Sure, we're waiting for a jwt and BE sign-in route implementation and then will ask @h0wter to adapt ProtectedRoute